test/functional/vector_tile_3d_mesh_mapbox.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('assert');
+
+describe('vector_tile_3d_mesh_mapbox', function _() {
+    let result;
+    before(async () => {
+        result = await loadExample('examples/vector_tile_3d_mesh_mapbox.html', this.fullTitle());
+    });
+
+    it('should run', async () => {
+        assert.ok(result);
+    });
+
+    it('load features data', async function _2() {
```

Comment:
rename to `should correctly load features at given coordinates`

Reply:
Corrected